lib/metasploit/framework/login_scanner/octopusdeploy.rb

```diff
@@ -33,18 +33,17 @@ def attempt_login(credential)
           else
             result_opts[:service_name] = 'http'
           end
-          begin
-            json_post_data = JSON.pretty_generate({ Username: credential.public, Password: credential.private })
-            cli = Rex::Proto::Http::Client.new(host, port, { 'Msf' => framework, 'MsfExploit' => framework_module }, ssl, ssl_version, http_username, http_password)
-            configure_http_client(cli)
-            cli.connect
-            req = cli.request_cgi(
+          begin 
+            res = send_request({
               'method' => 'POST',
               'uri' => uri,
               'ctype' => 'application/json',
-              'data' => json_post_data
-            )
-            res = cli.send_recv(req)
+              'vars_post' => {
+                'Username' => credential.public,
+                'Password' => credential.private
+              }
```

Comment:
This looks like it should be JSON data and I'm not sure this format will be correct? Might want to check that is the same output as JSON.pretty_generate.